misc/testerina/modules/testerina-core/src/main/java/org/ballerinalang/testerina/core/BTestRunner.java

```diff
@@ -440,18 +465,41 @@ private void execute(boolean buildWithTests) {
             // Call module stop and test stop function
             suite.getInitFunction().invokeStopFunctions();
         });
+
+        // Coverage report handling
+        if (!buildWithTests && coverageFlag) {
+
+            try {
+
```

Comment:
Unnecessary newlines.

Reply:
Removed 1 newline not both because of readability